CONTRIBUTING.md

```diff
@@ -487,71 +487,71 @@ It's mandatory to use the "feature/" prefix in the branch name.
 Example:
 
 
-* Create branch
+* Create branch:
 
 ```sh
 cd mapbender
 git checkout -b "feature/mega-cool-feature-x"
 ``` 
-* Improve the code
-* Save changes 
+* Improve the code.
+* Save changes: 
 
 ```sh
 git add *
 git commit -m "Add some new stuff"
 ``` 
-* Merge current release code
+* Merge current release code:
 
 ```sh
 git fetch -a
 git merge "release/3.0.6"
 ``` 
-* If conflicts arise, resolve [them][Resolve git conflicts]
-* Run tests
-* Push the changes on [github]
+* If conflicts arise, resolve [them][Resolve git conflicts].
+* Run tests.
+* Push the changes on [github]:
 
 ```sh
 git push
 ``` 
-* Create [pull-request]
+* Create [pull-request]:
 
-Then please wait for the feedback. We will check it out and review your code to merge it in the branch.
+Then just wait for our feedback. We will check it out and review your code to merge it in the branch. Thanks!
 
 ## Bug fix branch
 
-It's mandatory to use "hotfix/" prefix in your branch name.
+It's mandatory to use the "hotfix/" prefix in your branch name.
```

Comment:
Do we distinguish between bugfix and hotfix? I thought, hotfix means immediate new release with one fix and for bugfixes, we collect some and then release a new version.

Reply:
Yepp. Will change it. The small details are OK.